aptos-move/framework/aptos-framework/sources/aptos_account.move

```diff
@@ -40,6 +40,18 @@ module aptos_framework::aptos_account {
         coin::register<AptosCoin>(&signer);
     }
 
+    /// Batch version of APT transfer.
+    public entry fun batch_transfer(source: &signer, recipients: vector<address>, amounts: vector<u64>) {
+        let i = 0;
```

Comment:
do we want to assert the vector lengths are the same?

Reply:
Updated. It is better to fail explicitly rather than discarding the extra recipients or amounts.